Formula/gap.rb

```diff
@@ -0,0 +1,137 @@
+class Gap < Formula
+  desc "System for computational discrete algebra"
+  homepage "https://www.gap-system.org/"
+  url "https://www.gap-system.org/pub/gap/gap48/tar.bz2/gap4r8p9_2017_12_18-23_44.tar.bz2"
+  version "4.8.9"
+  sha256 "4c5a5bbbdb5275213cc575174f7bfb8bfd61c12bd082770798162b2ae6ac577d"
+
+  bottle do
+    cellar :any
+    sha256 "714ba2f943796f16be21d9dd7ac562873c481c38d8a47620f99b8571c19a25f7" => :high_sierra
+    sha256 "b745466b7b76c750b79d00f9ca6c47c154a0a2e18988caf4fcd8bd15c282697d" => :sierra
+    sha256 "fdaf283fb6c9657dc75e142b957759bc63c5b9a81f5545564df1b030ccb83564" => :el_capitan
+    sha256 "9eca000ecddafb1101edb06cd6087f87e38f9d168d5224e8c7030ccf6bf85853" => :x86_64_linux
+  end
+
+  # NOTE:  the archive contains the [GMP library](https://gmplib.org/) under
+  #   `extern/`, which is not needed if it is already installed (for example,
+  #   with Homebrew), and a number of GAP packages under `pkg/`, some of
+  #   which need to be built.
+
+  option "with-packages",
```

Comment:
Analytics say most users will install with `--with-packages`, so this should be made the default behaviour and the option should be removed.